Deepak_Yadav/tic tac toe/scripts.js

```diff
@@ -0,0 +1,170 @@
+var options = "X";
+//background color will change once any perfect combination occurs
+function highLight(cell1, cell2, cell3, symbol){
+    btnDisable();
+    document.getElementById(cell1).style.backgroundColor = 'red';
+    document.getElementById(cell2).style.backgroundColor = 'red';
+    document.getElementById(cell3).style.backgroundColor = 'red';
+}
+
+//game will start from starting with a new different background color
+function reset(){
+    var b1 = document.getElementById("b1");
```

Comment:
missing DRY KISS, extra code
code optimization required

Reply:
i was optimizing this yesterday..once completely optimized i will commit changes